aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc.md

```diff
@@ -1,202 +1,196 @@
 ---
 uid: signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc
-title: "Tutorial: Getting Started with SignalR 2 and MVC 5 | Microsoft Docs"
+title: "Real-time chat with SignalR 2 and MVC 5 | Microsoft Docs"
```

Comment:
Per the template, title should have the word Tutorial in it.

Reply:
@tdykstra, need some clarification on this one.

In [the User Story](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1375128), the 3rd bullet in the **Description** says, "H1 should begin “Tutorial: “ followed by a verb (“Tutorial” not in ToC entry or meta title)." Also, [the tutorial](https://review.docs.microsoft.com/en-us/help/contribute/templates/dotnet/tutorial-template-dotnet?branch=master) doesn't mention using the word "Tutorial" in the title. Has there been a change that I am unaware of? Am I reading the instructions wrong? Sorry, but I just want to make sure that I understand before I proceed.

Reply:
That requirement is in the global tutorial template, I didn't realize the .NET template doesn't have it. So for now consider "tutorial" in the metadata title to be an option that I'm suggesting for SEO, but not an absolute requirement of the template. I'll follow up to find out if the omission in the .NET template was on purpose.

Reply:
Thanks very much for the clarification. I added "Tutorial:" to the title. If you find out that the omission in the .NET template was on purpose, please let me know and I ill revert the change.

Reply:
Omitting that note from the .NET tutorial template was an oversight, and it's been updated to specify that "tutorial" should be in the metadata title.

Reply:
Thank you for the verification. I will make sure anyone else on my team working on this project knows about the omission.